extensions/spring-data-jpa/runtime/src/main/java/io/quarkus/spring/data/runtime/TypesConverter.java

```diff
@@ -35,6 +35,12 @@ private TypesConverter() {
     }
 
     public static io.quarkus.panache.common.Page toPanachePage(org.springframework.data.domain.Pageable pageable) {
-        return new io.quarkus.panache.common.Page(pageable.getPageNumber(), pageable.getPageSize());
+        int pageNumber = 0;
+        int pageSize = Integer.MAX_VALUE;
```

Comment:
I'm a bit worried that this will generate queries with paging all the time?

Especially in the context of https://github.com/quarkusio/quarkus/pull/8162/files coming.

Can we make sure this is not the case?

Reply:
Admitedly this isn't the perfect solution (ideally `toPanachePage` wouldn't be called at all when `Pageable` is `unpaged`).
I guess with #8162 coming (I hadn't seen that one), we should be take care of it now. @aureamunoz would you like to do that or do you want me to?

Reply:
Hi, yes, I will take a look.

Reply:
I just pushed another approach after look at the other PR (https://github.com/quarkusio/quarkus/pull/8162/files ). WDYT? @geoand @gsmet